### ISSUE TITLE
In some cases Micrometer doesn't expose `Meter`s `description` to `Prometheus`

### ISSUE THREAD

**Issue by yskopets** [micrometer-metrics/micrometer#159]
#### Issue

In some cases `Micrometer` doesn't expose `Meter`'s `description` to `Prometheus`

#### Steps to reproduce

```
MeterRegistry registry = new PrometheusMeterRegistry();
Meter.Id meterId = registry.createId("log_dropped_events", Tags.zip("appender", "stdout_drops"), "Total number of dropped log events");
registry.more().counter(meterId, new DoubleAdder(), DoubleAdder::sum);
```

#### Observed behaviour

```
# HELP log_dropped_events_total  
# TYPE log_dropped_events_total counter
log_dropped_events_total{appender="stdout_drops",} 0.0
```
Notice that `description` is missing

#### Expected behaviour

```
# HELP log_dropped_events_total  Total number of dropped log events
# TYPE log_dropped_events_total counter
log_dropped_events_total{appender="stdout_drops",} 0.0
```

#### Affected versions

v1.0.0-rc.2

**Comment by jkschneider** [comment]
Thanks again for the detailed report @yskopets.

**Comment by checketts** [comment]
Fixed by 2fb6c42
